tasks/deploy.yaml

```diff
@@ -45,6 +45,10 @@ spec:
       type: string
       description: "Extra arguments for the deployment"
       default: ""
+    - name: DEPLOY_FRONTENDS
+      type: boolean
```

Comment:
Type boolean doesn't exist in Tekton, these are the correct types: https://tekton.dev/docs/pipelines/tasks/#parameter-type